File: Android/sources/com/android/billingclient/api/zzv.java
package com.android.billingclient.api;

import java.util.concurrent.Callable;

public final /* synthetic */ class zzv implements Callable {
    public final /* synthetic */ BillingClientImpl zza;
    public final /* synthetic */ AlternativeBillingOnlyReportingDetailsListener zzb;

    public /* synthetic */ zzv(BillingClientImpl billingClientImpl, AlternativeBillingOnlyReportingDetailsListener alternativeBillingOnlyReportingDetailsListener) {
        this.zza = billingClientImpl;
        this.zzb = alternativeBillingOnlyReportingDetailsListener;
    }

    public final Object call() {
        this.zza.zzq(this.zzb);
        return null;
    }
}

File: IOS/v2/com/android/billingclient/api/zzv.swift
package com.iOS.billingclient.api;.swift
import java.util.concurrent.Callable;.swift
public final /* synthetic */ class zzv implements Callable {.swift
public final /* synthetic */ BillingClientImpl zza;.swift
public final /* synthetic */ AlternativeBillingOnlyReportingDetailsListener zzb;.swift
public /* synthetic */ zzv(BillingClientImpl billingClientImpl, AlternativeBillingOnlyReportingDetailsListener alternativeBillingOnlyReportingDetailsListener) {.swift
this.zza = billingClientImpl;.swift
this.zzb = alternativeBillingOnlyReportingDetailsListener;.swift
}.swift
public final Object call() {.swift
this.zza.zzq(this.zzb);.swift
return null;.swift
}.swift
}.swift
